Sources/SwiftKafka/RDKafka/RDKafkaConfig.swift

```diff
@@ -13,17 +13,18 @@
 //===----------------------------------------------------------------------===//
 
 import Crdkafka
+import Logging
 
 /// A collection of helper functions wrapping common `rd_kafka_conf_*` functions in Swift.
 struct RDKafkaConfig {
     /// Wraps a Swift closure inside of a class to be able to pass it to `librdkafka` as an `OpaquePointer`.
-    final class CapturedClosure {
+    final class Opaque {
```

Comment:
I'm still unsure about this one — generally, I think the logging callback should follow the same pattern as the delivery report callback (meaning capturing a Swift closure and invoking it from inside the C closure).

My idea for this case would be to have something like this:

```swift
final class CapturedClosures {
    let callbackClosure: ...
    let loggingClosure: ...
}
```

Then instead of having `setDeliveryCallback` and `setLoggingCallback` we would invoke something like `registerCallbacks(deliveryReportCallback:, loggingCallback:)` (best name I could think of) inside of `RDKafkaConfig` taking two Swift closures, wrapping them in C closures and then doing the dance with `rd_kafka_conf_set_dr_cb` and `rd_kafka_conf_set_log_cb`.

We can then just capture the `logger` inside of the Swift closure we passed here.

cc: @FranzBusch